scalalib/src/mill/javalib/android/AndroidAppModule.scala

```diff
@@ -62,6 +88,32 @@ trait AndroidAppModule extends JavaModule {
    */
   def androidReleaseKeyPath: T[PathRef] = Task.Source(millSourcePath / androidReleaseKeyName())
 
+  /**
+   * Specifies the file format of lint report. Available file formats are "html", "xml", "txt" and "sarif".
```

Comment:
nit: probably you would like to reference `AndroidLintReportFormat` values here instead of string literals